dashboard/assets/packages/ui/src/components/Chart/Chart.tsx

```diff
@@ -68,19 +75,18 @@ export default function Chart({ panel }) {
     options.axes[2].side = 1
   }
 
-  let select = theme.palette.mode == "dark" ? "#60606080" : "#d0d0d080"
+  function onCreate(chart: uPlot) {
+    const color = theme.palette.mode == "dark" ? "#60606080" : "#d0d0d080"
+    const element = chart.root.querySelector(".u-select") as HTMLElement
 
-  function onCreate(chart) {
-    chart.root.querySelector(".u-select").style.background = select
+    if (element) {
+      element.style.background = color
+    }
   }
 
   return (
     <Grid ref={ref} className="chart panel" item md={12} lg={6}>
-      <UplotReact options={options} data={plot.data} onCreate={onCreate} />
+      <UplotReact options={options} data={plot.data as AlignedData} onCreate={onCreate} />
```

Comment:
Perhaps we could find a way for SeriesPlot to return the correct type? `Type '(number | undefined)[][]' is not assignable to type 'AlignedData'.`